.pre-commit-config.yaml

```diff
@@ -19,25 +20,22 @@ repos:
       - id: check-toml
       - id: end-of-file-fixer
       - id: trailing-whitespace
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        name: isort (python)
+        args: ["--profile", "black", "--filter-files"]
   - repo: https://github.com/pycqa/flake8
     rev: 5.0.4
     hooks:
       - id: flake8
         additional_dependencies:
           - flake8-bugbear
           - flake8-comprehensions
-          - flake8-tidy-imports
         args: [--max-line-length=88]
-  - repo: https://github.com/asottile/reorder_python_imports
```

Comment:
Removing this since it is already handled by `isort`

Reply:
This is not entirely true - isort does not seperate imports into multiple lines: https://github.com/asottile/reorder_python_imports#why-this-style

I vote we replace isort with reorder_pyton_imports.